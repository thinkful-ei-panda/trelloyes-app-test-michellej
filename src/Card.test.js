import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Card from './Card';

const props = {
  key: 'id',
  title: 'title',
  content: 'lorem'
}

describe('Card Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Card {...props} />, div)
    ReactDOM.unmountComponentAtNode(div);
  });

  it('displays title and content', () => {
    const tree = renderer.create(
      <Card title='title' content='lorem' />
    ).toJSON()
    expect(tree).toMatchSnapshot();
  });
  
})