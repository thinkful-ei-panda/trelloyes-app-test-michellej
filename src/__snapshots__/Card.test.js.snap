// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Component displays title and content 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    title
  </h3>
  <p>
    lorem
  </p>
</div>
`;
